modules/imgproc/src/hough.cpp

```diff
@@ -1210,10 +1210,11 @@ icvHoughCirclesGradient( CvMat* img, float dp, float min_dist,
         // Check if the circle has enough support
         if( max_count > acc_threshold )
         {
-            float c[3];
+            float c[4];
```

Comment:
These changes have no effect because `cvSeqPush` copies only first 3 elements of `c`. To fix this make sufficient changes [here](https://github.com/opencv/opencv/pull/8909/files#diff-f3240fa1ccc6896a4c9e16db598360f0L1260) and [here](https://github.com/opencv/opencv/pull/8909/files#diff-f3240fa1ccc6896a4c9e16db598360f0L1272).